ignite/cmd/model/chain_serve.go

```diff
@@ -261,6 +261,9 @@ func (m ChainServe) renderQuitView() string {
 
 	fmt.Fprintf(&view, "%s %s\n", icons.Info, colors.Info("Stopped"))
 
+	// Survey Link
```

Comment:
It would be great if this could fetch an url and render it, for instance `api.ignite.com/announcements` and if it fails, render a static url (should be a constant, somewhere else)

Reply:
sorry, render it as in render the html / json contents of the url on the terminal or check if the url is returning 200 then show that url or show a static url?

Reply:
Parsing some json and printing out the values nicely. Something like `{ "announcements": [ "New Survey Available at https://example.com", "Something Else"]}`

Reply:
sounds like a good idea, do you want to keep that part of this pr or want me to make a new one?

Reply:
Doesn't matter, whatever your prefer. You could update this PR too